recipes/python-patch/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "patch" %}
+{% set version = "1.16" %}
+{% set sha256 = "c62073f356cff054c8ac24496f1a3d7cfa137835c31e9af39a9f5292fd75bd9f" %}
+
+package:
+  name: python-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - patch
+
+about:
+  home: https://github.com/techtonik/python-patch/
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
```

Comment:
Use quotes around the file path. `license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'`

Reply:
Wow, why is this needed? It appears to work though.

Reply:
@windelbouwmanbosch It's a YAML syntax thing, unfortunately. Otherwise it sees the `{` and thinks you're trying to write a dictionary.